scripts/build.py

```diff
@@ -156,29 +226,99 @@ def process_js(source_path, target_path):
     _minify(source_path, target_path)
 
 
-def build_minified_third_party_libs(output_directory):
+def get_dependency_filepaths(dependency, filepaths):
+    """Extracts dependency filepaths into filepaths object.
+
+    Args:
+        source_paths: dict(str, str). Dependency information object.
```

Comment:
Can you talk more about how this is formatted? What do the keys and values mean?

Reply:
Also, arg name does not match function signature.

Reply:
Done.